helm/monitoring/templates/grafana/deployment.yaml

```
@@ -0,0 +1,33 @@
+    matchLabels:
+      app: grafana
+  template:
+    metadata:
+      labels:
+        app: grafana
+    spec:
+      containers:
+        - name: grafana
+          image: "{{ .Values.grafana.image.repository }}:{{ .Values.grafana.image.tag }}"
+          ports:
+            - containerPort: {{ .Values.grafana.service.port }}
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)